### PR TITLE
Adds test for widemul, div and rem in int.jl

### PR DESCRIPTION
**Pull request by mronian**

### CODE REVIEW

File: test/int.jl [hunk 1]
```diff
@@ -176,6 +176,19 @@ end
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
 
+i=2^63-1
```
Comment (tkelman) [inline test/int.jl]: Note that `2^63` overflows on 32 bit. Use `Int64(2)^63` or similar

**Comment by tkelman** [comment]
@mronian I hope you plan on reopening this from a new branch? It's usually better to work from branches other than master when making a PR.

**Comment by mronian** [comment]
I made the changes and amended my commit but it didn't get added to this PR automatically. What did I do wrong? Will it work if I push the commit to a new branch?

**Comment by mronian** [comment]
I got confused since the PR closed after I amended my commit, but understood how this works after re-opening it. I'll open this PR from a new branch.

**Comment by mronian** [comment]
I've pushed the commit to a new branch. How do I re-open this PR from that branch?

**Comment by yuyichao** [comment]
You can't change the branch corresponding to a PR. Open a new one.

**Pull request closed without merging**